File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashSet<Integer> res = new HashSet<Integer>();
        for (int i = 0; i < nums.length; i++){
            if(res.contains(nums[i])){
                return true;
            }
            else{
                res.add(nums[i]);
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/pow-x-n/submission-0.cpp
class Solution {
public:
    double myPow(double x, int n) {
        double res = x;
        if (n < 0){
            int pow = x;
            for (int i = n; i <= 0; i++){
                pow *= x;
                res /= x;
            }
            return res;
        }
        else if (n > 0){
            for (int i = 1; i < n; i++){
                res *= x;
            }
            return res;
        }
        else return 1;
    }
};
